src/utilities.js

```diff
@@ -86,6 +86,26 @@ export function formatResolution(resolution = '') {
   return '';
 }
 
+export function formatKeywords(obj) {
+  var comparisonItems = [obj];
+
+  // If we are requesting an array of keywords for comparison
+  if (Array.isArray(obj.keyword)) {
```

Comment:
Having `comparisonItems` variable is kind of unnecessary.  Maybe instead start with the `if (Array.isarray(obj.keyword))` check and inside you can simply return the reduce.